### PR TITLE
371 cluster api

### PR DESCRIPTION
**Pull request by rigazilla**
Fix for #371

### CODE REVIEW

File: pkg/controller/infinispan/infinispan_controller.go [hunk 1]
```diff
@@ -793,7 +794,15 @@ func existsCacheServiceDefaultCache(podName string, infinispan *infinispanv1.Inf
 	namespace := infinispan.ObjectMeta.Namespace
 	secretName := infinispan.GetSecretName()
 	protocol := infinispan.GetEndpointScheme()
-	return cluster.ExistsCache("default", secretName, podName, namespace, string(protocol))
```
Comment (dmvolod) [inline pkg/controller/infinispan/infinispan_controller.go:796]: @rigazilla what do you think to create a constant for 'default' cache name?

File: pkg/controller/infinispan/infinispan_controller.go [hunk 2]
```diff
@@ -793,7 +794,15 @@ func existsCacheServiceDefaultCache(podName string, infinispan *infinispanv1.Inf
...
+	pass, err := kubernetes.GetPassword(consts.DefaultOperatorUser, secretName, namespace)
+	if err != nil {
+		return false
+	}
+	res, err := cluster.ExistsCache(consts.DefaultOperatorUser, pass, "default", podName, namespace, string(protocol))
```
Comment (dmvolod) [inline pkg/controller/infinispan/infinispan_controller.go:801]: @rigazilla what do you think to create a constant for 'default' cache name?

File: test/e2e/main_test.go [hunk 3]
```diff
@@ -373,12 +373,14 @@ func waitForPodsOrFail(spec *ispnv1.Infinispan, num int) {
...
+	pass, err := cluster.Kubernetes.GetPassword("operator", spec.GetSecretName(), tconst.Namespace)
+	testutil.ExpectNoError(err)
+
 	expectedClusterSize := num
 	// Check that the cluster size is num querying the first pod
 	var lastErr error
-	err := wait.Poll(time.Second, tconst.TestTimeout, func() (done bool, err error) {
-		value, err := cluster.GetClusterSize(secretName, podName, tconst.Namespace, protocol)
+	err = wait.Poll(time.Second, tconst.TestTimeout, func() (done bool, err error) {
+		value, err := cluster.GetClusterSize("operator", pass, podName, tconst.Namespace, protocol)
```
Comment (dmvolod) [inline test/e2e/main_test.go:383]: We can use constant here

**Comment by rigazilla** [comment]
merged

**Pull request merged**